src/EventList.js

```diff
@@ -74,12 +74,13 @@ export function EventList(props) {
     }
 
     return (
-      <a href={event['browser_url']} 
-        className="eventCard" 
-        target="_blank" 
-        key={event['id']} 
-        coord={('location' in event && 'location' in event['location'] && 'latitude' in event['location']['location']) ? "" + event['location']['location']['latitude'] + "&" + event['location']['location']['longitude'] : ""} 
-        onMouseEnter={(event) => { props.updatedHover(event['currentTarget'].getAttribute('coord')) }} 
+      <a href={event['browser_url']}
+        className="eventCard"
+        target="_blank"
+        rel="noopener noreferrer"
+        key={event['id']}
+        coord={('location' in event && 'location' in event['location'] && 'latitude' in event['location']['location']) ? "" + event['location']['location']['latitude'] + "&" + event['location']['location']['longitude'] : ""}
```

Comment:
Yeah, there are a ton of these. It’s hard to grok exactly what this long list of conditionals is checking. 

A class would really clean a lot of this up, or alternatively, a function that is evocatively named and accepts the event as an arg to replace these in-line conditionals